lib/json.go

```diff
@@ -0,0 +1,107 @@
+package lib
+
+import (
+	"encoding/json"
+)
+
+func DecodeFromJSON(in []byte, out any) error {
+	return json.Unmarshal(in, out)
+}
+
+func EncodeToJSON(in any) ([]byte, error) {
+	return json.Marshal(in)
+}
+
+func ParseJsonArray(value any) []any {
```

Comment:
nit: not very go like I think ? Might be better to Unmarshal to a go object ? (I am a bit allergic to `any`)

Reply:
Yeah. Those are vestiges of earlier attempts to deal with JSON in Go. I now know better and intend to get rid of the entire `json.go` file eventually.